CaseBonita/Services/PlaylistUploader/Handlers/Spotify.py

```diff
@@ -1,26 +1,77 @@
+import json
+import webbrowser
+
+import requests
 import spotipy as sp
 from spotipy.oauth2 import SpotifyClientCredentials
 
 from CaseBonita.Services.PlaylistUploader.Handlers.Base import BaseAPIUploader
-from .consts import CLIENT_ID, CLIENT_SECRET
+from .consts import CLIENT_ID, CLIENT_SECRET, SPOTIFY_USER_ACCESS
+from .utils import insert_id_to_url
 
 
 class UploadToSpotify(BaseAPIUploader):
     def __init__(self, developer_key, _songs_json_file, user_credentials):
         super(BaseAPIUploader, self).__init__(developer_key, _songs_json_file, user_credentials)
 
-    def create_developer_credentials(self):
+    @classmethod
+    def return_developer_credentials(cls):
```

Comment:
change the function name to `get_developer_key`

Reply:
for development, it's okay, if we want to be extra safe, In triangulation, I made a txt file that is password protected and another hidden in system one which contains the password and a script to read it (uses passlib to mask the path/password).
I don't think that should be of any concern right now, waiting to mongo is okay.

Reply:
Waiting for mongo it is then.